.gitignore

```diff
@@ -54,6 +54,7 @@ _testmain.go
 
 # Log files
 *mattermost.log
+*mattermost.log.jsonl
```

Comment:
Can we add this to `make clean`?

Reply:
Done in https://github.com/mattermost/mattermost-server/pull/7466/commits/aa9393b1627a5d9c79674c8d9bb7d0d7f2172918